src/Data/Protocol.cs

```diff
@@ -25,6 +25,7 @@ public MessageHeader()
         {
             ProtocolVersion = "5.2.0";
             Id = Guid.NewGuid().ToString();
+            Username = Environment.UserName;
```

Comment:
??

Reply:
Ah, that was a minor bug fix I forgot to document, sorry. The protocol requires that the username isn't blank, and we were technically violating JMP spec.